testapp/home/templates/home/hubs_page.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+
+  {% block main %}
```

Comment:
@sabi98 Mike is going to get you for this indenting

Reply:
👀 👿